jenkins/dev_env/conf/example_config.json

```diff
@@ -20,7 +20,12 @@
             "rack_id": "1",
             "storage_set_id": "storage-set-1",
             "s3_instances": "1",
-            "type": "VM"
+            "type": "HW",
```

Comment:
Will this break the deployment on VM?

Reply:
create another config called example_config_hw.json.

Reply:
Done